packages/util/src/generator/readme.ts

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
Generally I agree with reducing code duplication, but in this case I am not so sure. Up to now, all functionality from util is generic and could potentially also be used by our users. This is very specific and nobody apart from us will use it. In my opinion this is a constant, that is used in two places rather than a reusable function, that should be shared publicly.
I think as of today, I would rather duplicate the code in the two generators. If we encounter more of these specific code duplications, I would even consider creating a separate package for it. @FrankEssenberger what is your opinion?

Reply:
I think eventually we need a `generator-common` or `generator-util` package, for the time being, I would also like to prepare it. Therefore, a new folder is created in the `util`, which is called `generator` and will hold all the common functions. Without this, it's not clear when do we need a new package.

Reply:
Discussed with @FrankEssenberger , we will create a new package for that. I created a follow up.